src/components/layout/signed-out-user-navigation.jsx

```diff
@@ -18,7 +18,7 @@ export default function SignedOutUserNavigation(props) {
           onClose={props.toggleModal}
           showOauthModal={props.showOauthModal}
         />}
-      <MobileNavigationMenu />
+      <MobileNavigationMenu adminNavLink={props.adminNavLink} mobileNavList={props.mobileNavList} />
```

Comment:
doesn't hurt anything, but unsure of situation where `adminNavLink` would be present if signed out?

Reply:
It wouldn't, but I wanted to be consistent in supporting all the same props as the children since the only purpose of this is layout and nothing else.